jest.config.js

```diff
@@ -5,5 +5,14 @@ module.exports = {
     __testing__: true
   },
   roots: ['app/javascript'],
-  testMatch: ['**/*.test.js']
+  setupFiles: ['./config/jest.setup.js'],
+  testRegex: '(/__tests__/.*|(\\.|_|/)(test|spec))\\.(js?|ts?)$',
```

Comment:
`\.(js?|ts?)$` will match `.j` and `.js` but not `.jsx` ;)

I guess you wanted `\.(jsx?|tsx?)$`?

Reply:
You are totally right.